tools/rna_tools/inforna/inforna.xml

```diff
@@ -1,7 +1,7 @@
-<tool id="inforna" name="InfoRNA" version="0.1.0">
+<tool id="inforna" name="InfoRNA" version="2.1.2">
     <description></description>
     <requirements>
-        <requirement type="package" version="1.0">inforna</requirement>
+        <requirement type="package" version="2.1.2">inforna</requirement>
         <requirement type="package" version="2.1.5">vienna_rna</requirement>
```

Comment:
vienna_rna is not needed with conda as separate dependency https://github.com/bioconda/bioconda-recipes/blob/master/recipes/inforna/meta.yaml#L21